src/pages/AdminEditUser.jsx

```diff
@@ -1,4 +1,4 @@
-import _ from 'lodash';
+import {map as ldmap, concat, filter, matches as ldmatches} from 'lodash';
```

Comment:
Optional Nit: I find `ldmap` potentially confusing, how would you feel about just spelling out `lodashMap`? Same thing for matches.

Reply:
Good idea.